vagrant_support/checksetup_answers.j2

```diff
@@ -0,0 +1,37 @@
+$answer{'ADMIN_EMAIL'}          = 'vagrant@{{ WEB_HOSTNAME }}';
+$answer{'ADMIN_OK'}             = 'Y';
+$answer{'ADMIN_PASSWORD'}       = 'vagrant01!';
```

Comment:
Yeah, WONTFIX on that. This is not meant to be a production setup, but something that you can use to quickly diagnose problems and review patches.